core/controllers/reader.py

```diff
@@ -338,17 +330,18 @@ def post(self, exploration_id):
                         customization_args[issue_keyname]):
                     if (len(issue['playthrough_ids']) <
                             feconf.MAX_PLAYTHROUGHS_FOR_ISSUE):
+                        playthrough_id = (
+                            stats_models.PlaythroughModel.create_from_dict(
+                                playthrough_data))
                         exp_issues.unresolved_issues[index][
                             'playthrough_ids'].append(playthrough_id)
-                    else:
-                        # In this case, the playthrough instance created doesn't
-                        # need to be stored, so it is deleted.
-                        stats_models.PlaythroughModel.get(
-                            playthrough_id).delete()
                     issue_found = True
```

Comment:
nit: consider moving this statement up to line 331, since it is at that point in the logic that you are certain that the issue has been found. Easier to read and follow the code that way.